### PR TITLE
Upgrade app to .NET Core 3.1 and libraries to .NET Standard 2.1

### PR DESCRIPTION
**Pull request by chadsowald**
There will be a subsequent PR to swap up JSON.NET with the newer System.Text.Json library.

### CODE REVIEW

**Comment by jasmcn** [comment]
`error NETSDK1045: The current .NET SDK does not support targeting .NET Standard 2.1`

Seeing that in the build; guessing the build pipeline doesn't support .net core 3.x yet?

File: src/Infrastructure/Data/CatalogContext.cs [hunk 1]
```diff
@@ -74,7 +73,7 @@ private void ConfigureCatalogBrand(EntityTypeBuilder<CatalogBrand> builder)
             builder.HasKey(ci => ci.Id);
 
             builder.Property(ci => ci.Id)
-               .ForSqlServerUseSequenceHiLo("catalog_brand_hilo")
+               .UseHiLo("catalog_brand_hilo")
```
Comment (jasmcn) [inline src/Infrastructure/Data/CatalogContext.cs:76]: Is the old method deprecated? What's the difference between this and the old method?

File: src/Web/Web.csproj [hunk 2]
```diff
@@ -1,21 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>Contoso.FraudProtection.Web</RootNamespace>
-    <UserSecretsId>95ed0ac6-6749-4920-aaf4-6f92323204d4</UserSecretsId>
-    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
```
Comment (jasmcn) [inline src/Web/Web.csproj:7]: HostingModel no longer a thing in 3.1?

File: src/Infrastructure/Data/CatalogContext.cs (hunk 1 above)
Comment (chadsowald) [reply src/Infrastructure/Data/CatalogContext.cs:76]: Yes and no difference.  I think they just renamed it as far as I can tell from docs/recommendations.

**Comment by chadsowald** [comment]
https://docs.microsoft.com/en-us/dotnet/core/compatibility/2.2-3.1

File: src/Web/Web.csproj (hunk 2 above)
Comment (chadsowald) [reply src/Web/Web.csproj:7]: https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#in-process-hosting-model  It's the default now

**Comment by chadsowald** [comment]
@jasmcn Updated the build pipeline file too.  Build is now passing.

**Pull request merged**